### PR TITLE
feat(natives/ped): add SetPedPhonePaletteIdx

### PR DESCRIPTION
**Pull request by ahcenezdh**
For js & c# examples i'm not sure at all, someone have to review it to be sure. Close the issue #989

### CODE REVIEW

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, this replaces a native, therefor the old [native file](https://github.com/citizenfx/natives/blob/f1f7a7d3f438da81d006bf0dfb59012d40c8272e/PED/N_0x83a169eabcdb10a2.md?plain=1#L4) should be removed. Ideally, an alias pointing to the old native name should be added as well since that's standard.

File: PED/SetPedPhonePaletteIdx.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+```c
+// 0x83A169EABCDB10A2
+void SET_PED_PHONE_PALETTE_IDX(Ped ped, int index);
+```
+
+Sets the palette index of a ped's phone.
+
+| Value | Color   |
+| :---: | :-----: |
+|  `0`  | Blue    |
```
Comment (PsychoShock) [inline PED/SetPedPhonePaletteIdx.md:15]: Actually `0` is light Blue theme from the settings menu.

**Comment by ahcenezdh** [comment]
> Hey there, this replaces a native, therefor the old [native file](https://github.com/citizenfx/natives/blob/f1f7a7d3f438da81d006bf0dfb59012d40c8272e/PED/N_0x83a169eabcdb10a2.md?plain=1#L4) should be removed. Ideally, an alias pointing to the old native name should be added as well since that's standard.

For x reason when searching it on my visual studio code, couldn't find it

File: PED/SetPedPhonePaletteIdx.md (hunk 1 above)
Comment (ahcenezdh) [reply PED/SetPedPhonePaletteIdx.md:15]: 👍

**Comment by ahcenezdh** [comment]
Should be good

**Comment by ahcenezdh** [comment]
Hey @4mmonium srry to ping you but any news for this PR?

File: PED/SetPedPhonePaletteIdx.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        print("Color not found: " .. color) -- Color doesn't exist in the table. 
+        return -- Exits the function.
+    end
+
+    -- Use the native to set the ped's phone palette index based on the selected color.
+    SetPedPhonePaletteIdx(playerPed, colors[color])
+end
+
+-- Call the function with "Green" to set the phone color to green for the player
+setPhoneColor("Green")
```
Comment (4mmonium) [inline PED/SetPedPhonePaletteIdx.md:61]: This is neat, but it just complicates the example for the average newcomer. Don't introduce logic that could result complex for newcomers, simply define the `enum` for the colors somewhere and refer to the `enum`. Same applies for the other examples. 

This is great, but why doing this when the native takes an integer anyway? Both would achieve the same result, just less complex and more straight to the point.

Thanks!

**Review by 4mmonium — CHANGES_REQUESTED** [review]
This is OK, but I'd prefer if you used an `enum` rather than passing it as a `string` for hash table (key-value pair tables) lookup. Check my comment, thanks!

**Review by 4mmonium — APPROVED** [review]
Looks good to me, merging, thank you!

**Pull request merged**